app/models/puppetdb_client.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'puppetdb'
+
+class PuppetdbClient
+  def initialize(endpoint)
+    @endpoint = endpoint
+    @client = PuppetDB::Client.new(server: @endpoint)
+  end
+
+  def ipaddress(fqdn)
+    response = @client.request('facts', ['and', ['=', 'certname', fqdn], ['=', 'name', 'ipaddress']])
```

Comment:
Perhaps we can set an `attr_reader :client` and therefore just use `client` here?